cloudferrylib/os/compute/nova_compute.py

```diff
@@ -658,7 +658,7 @@ def deploy_instance(self, create_params, conf):
                 self.wait_for_status(new_id, self.get_status, 'active',
                                      timeout=conf.migrate.boot_timeout,
                                      stop_statuses=[ERROR])
-            except exception.TimeoutException:
+            except (exception.TimeoutException, KeyboardInterrupt):
```

Comment:
I think your do not understand the main idea. For me, a successful instance - is the one that has reached the point checkpointvm